### ISSUE TITLE
Uncaught exception

### ISSUE THREAD

**Issue by jasonbronson** [hovancik/stretchly#400]
Uncaught Exception:
TypeError: Cannot read property 'length' of null
    at closeWindows (/Applications/stretchly.app/Contents/Resources/app.asar/main.js:111:28)
    at breakComplete (/Applications/stretchly.app/Contents/Resources/app.asar/main.js:433:3)
    at finishBreak (/Applications/stretchly.app/Contents/Resources/app.asar/main.js:447:3)
    at Function.globalShortcut.register (/Applications/stretchly.app/Contents/Resources/app.asar/main.js:372:9)

Using latest version window popped up and I was using vscode to code some items when error showed up.

**Comment by hovancik** [comment]
Hi and congratz on issue # 400! ;] 
 
Do you use more monitors or one? Was it at the end of break?

**Comment by lkrms** [comment]
Almost identical (except for finishBreak vs. postponeMicrobreak):

![DeepinScreenshot_stretchly_20190905042031](https://user-images.githubusercontent.com/2563783/64280564-b6c53e00-cf94-11e9-9610-21265aa40ba6.png)

In my case (and I suspect it's the same in yours), it arises only when I click "Reset breaks" from the tray menu _while a stretchly window is open_. I've found that it's because stretchly doesn't release the `CommandOrControl+X` global shortcut when windows are closed this way. I've tested a fix here. Preparing a PR.
